staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -211,7 +211,7 @@ type Interface interface {
 	//       // a uint64 to alter the TTL on the object, or nil to keep it the same value.
 	//       return cur, nil, nil
 	//    }
```

Comment:
it needs a "," here

Reply:
why?

Reply:
it is go syntax I guess when you close function parentheses on the next line, all your lines need to have ,.

[this](https://play.golang.org/p/U7W5eYvJ1JL) vs [this](https://play.golang.org/p/3ANYaBuHMWT)

Reply:
thank you